bulbs/poll/models.py

```diff
@@ -126,7 +126,12 @@ def get_sodahead_data(self):
         response = requests.get(SODAHEAD_POLL_ENDPOINT.format(self.sodahead_id))
 
         if not response.ok:
-            raise SodaheadResponseError(response.text)
+            return {
```

Comment:
This is slightly better: https://github.com/theonion/django-bulbs/blob/master/bulbs/content/models.py#L145